discordia/containers/snowflakes/Guild.lua

```diff
@@ -386,6 +386,36 @@ local function findMembers(self, predicate)
 	return self._members:findAll(predicate)
 end
 
+-- embed --
+
+local function getEmbed(self)
```

Comment:
It returns one object; it has got a singular name since you only pass One Embed Object to this class.
Of course a message can have more than one Embed but in this case you are not passing the whole message object and checking then for Embeds.